main.go

```diff
@@ -131,6 +146,87 @@ func runGradleTask(cmdFactory command.Factory, logger log.Logger, workDir string
 	return cmd.Run()
 }
 
+// exportTestResults finds all files matching pattern **/build/test-results/test*/TEST-*.xml
+func exportTestResults(projectRootDir string, taskStartTime, taskFinishTime time.Time, testResultsDir string, logger log.Logger, exporter output.Exporter) error {
```

Comment:
Does it matter for the glob to match wether projectRootDit is absolute or relative?